templates/web/base/admin/report-assignment.html

```diff
@@ -0,0 +1,7 @@
+[%  IF c.user.has_permission_to('report_inspect', c.cobrand.body.id ) %] 
```

Comment:
I think anyone who can see the report in backend admin interface should be able to see this, no need for a permission check.

Reply:
I am using the same template in admin and in the frontend reports list. I am moving out of 'admin' as part of the changes relating to your earlier comment on that template.